qleet/utils/circuit.py

```diff
@@ -0,0 +1,136 @@
+import typing
```

Comment:
Do you think it would improve readability if we do `from typing import x` instead of writing `typing.x` everytime?

Reply:
We can change to that.